### ISSUE TITLE
Summary should be printed after JKube actions

### ISSUE THREAD

**Issue by sunix** [eclipse/jkube#1033]
### Description

Globally the feeling I get is that we are not telling enough about what is JKube doing. Zero config magic is good, but as a developer I don’t feel confident if I don’t know what it is doing.
- [ ] Lists exactly what points should be summarized. Make sure that for any configuration combination, the summary is printed accordingly.
- [ ] Summary should be printed after the build/phase execution

**Comment by sunix** [comment]
Good point, in my opinion it should be printed too for the rest of actions

**Comment by sunix** [comment]
build:
- build strategy: docker/jib/s2i
- packaged file location (war or jar)
- generator name
- if dockerfile used: dockerfile location, otherwise: from, cmd, args, etc...
- target image name
- success or failure, if failure provide the cause

**Comment by rohanKanojia** [comment]
I've created this document for possible points to print in different jkube summary actions. Most of these are already covered in https://github.com/eclipse/jkube/pull/1686
https://docs.google.com/document/d/1x2z96UEeEezIS3uB8la9EdJVRiWhm4reHe2R1-9CAvo/edit?usp=sharing

**Comment by sunix** [comment]
Maybe we could add few concrete example of the expected output (with a concrete project) ? that could be than be reused in our integration tests ?
